tests/unit/src/views/auth/Login.test.js

```diff
@@ -0,0 +1,27 @@
+import Login from '@/views/auth/Login.vue';
+
+import testHelpers from '@@/testHelpers.js';
+
+describe('Login.vue', () => {
+  let localVue;
+  let store;
+  let setup;
+
+  let permissions = ['student'];
+
+  beforeEach(() => {
+    setup = testHelpers.setupTestVariables(permissions);
+    localVue = setup.localVue;
+    store = setup.store;
+  });
+
+  test('Renders correctly', async () => {
+    const wrapper = await testHelpers.shallowMountWrapper(Login, {
+      localVue,
+      store
+    });
+
+    expect(wrapper)
+      .toMatchSnapshot();
+  });
```

Comment:
need to add a test here for mocking login and verifying router.replace was called via spy.